src/components/Capture/Document.tsx

```diff
@@ -55,13 +58,11 @@ type Props = StepComponentDocumentProps &
   WithCaptureVariantProps
 
 class Document extends Component<Props> {
-  static defaultProps: Partial<Props> = {
+  static defaultProps = {
     forceCrossDevice: false,
-    requestedVariant: 'standard',
-    side: 'front',
   }
 
-  handleCapture: HandleCaptureProp = ({ variant, ...payload }) => {
+  handleCapture: HandleCaptureProp = (payload) => {
```

Comment:
This is a really great spot because in the latest feature branch, there'll be another `handleMultiFrameCapture`: https://github.com/onfido/onfido-sdk-ui/blob/feature/image-quality-enhancements/src/components/Capture/Document.tsx#L136 :joy: